scripts/safari.js

```diff
@@ -6997,6 +6944,128 @@ function Safari() {
             }
         }
     };
+    FactionWar.prototype.setupEvent = function() {
+        //SETUP TEAMS
+        var teamSize = Math.ceil(Math.max(this.signups.length, 6)/2);
+        
+        var pickedSides = this.preferredTeams, name, e, overflow = [], changedTeams = {};
+        for (e in pickedSides) {
```

Comment:
The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.